markdown2ctags.py

```diff
@@ -184,29 +186,29 @@ def genTagsFile(output, tags, sort):
 def main():
     from optparse import OptionParser
 
-    parser = OptionParser(usage = "usage: %prog [options] file(s)",
-                          version = __version__)
+    parser = OptionParser(usage="usage: %prog [options] file(s)",
+                          version=__version__)
     parser.add_option(
-            "-f", "--file", metavar = "FILE", dest = "tagfile",
-            default = "tags",
-            help = 'Write tags into FILE (default: "tags").  Use "-" to write '
-                   'tags to stdout.')
+        "-f", "--file", metavar="FILE", dest="tagfile",
+        default="tags",
+        help='Write tags into FILE (default: "tags").  Use "-" to write '
+        'tags to stdout.')
     parser.add_option(
-            "", "--sort", metavar="[yes|foldcase|no]", dest = "sort",
-            choices = ["yes", "no", "foldcase"],
-            default = "yes",
-            help = 'Produce sorted output.  Acceptable values are "yes", '
-                   '"no", and "foldcase".  Default is "yes".')
+        "--sort", metavar="[yes|foldcase|no]", dest="sort",
+        choices=["yes", "no", "foldcase"],
+        default="yes",
+        help='Produce sorted output.  Acceptable values are "yes", '
+        '"no", and "foldcase".  Default is "yes".')
 
     options, args = parser.parse_args()
 
     if options.tagfile == '-':
         output = sys.stdout
     else:
-        output = open(options.tagfile, 'wb')
+        output = open(options.tagfile, 'w')
 
     for filename in args:
-        f = open(filename, 'rb')
+        f = open(filename, 'r')
```

Comment:
Work well under python2, but not python3 in this case. 
```
 ✘ lsm@lsm-pc  /tmp  ~/.vim/bundle/markdown2ctags/markdown2ctags.py -f - a.md
Traceback (most recent call last):
  File "/home/lsm/.vim/bundle/markdown2ctags/markdown2ctags.py", line 223, in <module>
    main()
  File "/home/lsm/.vim/bundle/markdown2ctags/markdown2ctags.py", line 212, in main
    lines = f.read().splitlines()
  File "/usr/lib/python3.5/codecs.py", line 321, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xc9 in position 43: invalid continuation byte
```
I will debug it later ...